### PR TITLE
applied styling to all elements and tweaked positioning

### PR DESCRIPTION
**Pull request by PaulHowley**
Tweaked some of html code for the search buttons and styled all elements to look as close to the original google homepage as possible.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -25,26 +25,26 @@
...
-            <li><a class="lucky" href="#">I'm Feeling Lucky</a></li>
-        </ul>
     </form>
+    <div class="search-buttons">
+        <input value="Google Search" type="submit" class="google-search">
+        <input value="I'm Feeling Lucky" type="submit" class="lucky">
+    </div>
     <div class="uk-footer">
-        <p>United Kingdom</p>
+        <span class="UK">United Kingdom</span>
```
Comment (krisquigley) [inline index.html]: I would use a more generic class name for these as it shouldn't matter what the actual country is.

File: index.html [hunk 2]
```diff
@@ -25,26 +25,26 @@
...
     </div>
     <div class="main-footer">
         <div class="left-footer">
             <ul>
-                <li><a href="#">Advertising</a></li>
-                <li><a href="#">Business</a></li>
+                <li><a href="#" class="lf">Advertising</a></li>
```
Comment (krisquigley) [inline index.html]: Try to use more descriptive class names, if you come back to this project in a few weeks seeing `lf` and `rf` is not going to mean anything, or to other developers

**Review by krisquigley — COMMENTED** [review]
Looking good. Could you add some screenshots so I can see how it is looking please.

Also, I would update visual studio code to use 2 spaces for indentation and then update the indentation accordingly for both the CSS and HTML

File: index.html (hunk 1 above)
Comment (PaulHowley) [reply index.html]: Hi Kris, amendments made and screenshot included below:
image
Thanks.

**Comment by PaulHowley** [comment]
Full styling applied and tidied up html and css documents with indentation and more descriptive class names.

**Pull request merged**